app/src/main/java/unipd/se18/ocrcamera/ImageProcessing.java

```diff
@@ -1,20 +1,20 @@
-package unipd.se18.ocrcamera;
-
+package unipd.se18.ocrcamera;                  // Reviewed by Balzan Pietro, to avoid confusion my comments are indicated by ***
+                                               // both at the start and at the end
 import android.graphics.Bitmap;
 import android.os.Environment;
-import android.util.Log;
-
-import org.opencv.core.Core;
-import org.opencv.core.Mat;
+import android.util.Log;                        //***from the StyleDoc:
```

Comment:
from the StyleDoc: If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.